src/Model/Callback/Message.php

```diff
@@ -127,6 +142,13 @@ public static function create(array $callbackData)
         $quickReply = $callbackData['quick_reply']['payload'] ?? null;
         $attachments = $callbackData['attachments'] ?? [];
 
-        return new static($callbackData['mid'], $callbackData['seq'], $text, $quickReply, $attachments);
+        return new static(
+            $callbackData['mid'],
+            $callbackData['seq'],
+            isset($callbackData['nlp']) ? $callbackData['nlp']['entities'] : null,
```

Comment:
if `$callbackData['nlp']['entities']` doesn't exist, it should returns an `array`, not `null`. Can you also extract this logic inside a variable `$entities`, like `$quickReply` or `$attachments`